docs/index.md

```diff
@@ -10,11 +10,17 @@ description: >-
 
 import SupportedLanguagesTable from '/src/components/reference/_supported-languages-table.mdx'
 import MoreHelp from "/src/components/MoreHelp"
+import ThemedImage from '@theme/ThemedImage'
 
 <br />
 <p align="center">
   <a href="https://semgrep.dev">
-    <img src="https://raw.githubusercontent.com/returntocorp/semgrep/develop/semgrep.svg" height="100" alt="Semgrep logo" />
+    <ThemedImage
+      alt="Semgrep themed logo"
+      sources={{
+        light: ('https://raw.githubusercontent.com/returntocorp/semgrep/develop/semgrep.svg'),
```

Comment:
@adamkvitek Can you make this file a local asset as well?

Reply:
@pabloest done in: https://github.com/returntocorp/semgrep-docs/pull/881/commits/a1056ba72f2d787aa686ad2797ba061aff796c92 Please check :-)